test/functional/sqlsrv/MsHelper.inc

```diff
@@ -211,6 +214,27 @@ function getCekName()
 }
 
 /**
+ * @return the path to the KSP dll/so file
+ */
+function getKSPpath()
+{
+    $name = 'myKSP';
+
+    $dir_name = realpath(dirname(__FILE__));
+    $ksp = $dir_name . DIRECTORY_SEPARATOR . $name;
+    if ( strtoupper( substr( php_uname( 's' ), 0, 3 ) ) == 'WIN' ) {
```

Comment:
be consistent with coding standard (i.e., no space after open parenthesis and before close parenthesis)

Reply:
I know I must have missed some... too bad the editor no longer works for me :(

Reply:
:( hope you get it working soon.